src/assets/styles/20-tools/_extends/_lists.scss

```diff
@@ -35,8 +35,8 @@
 
   ol,
   ul {
-    margin-bottom: 0;
-    margin-top: var(--space-unit);
+    margin-bottom: var(--space-md);
```

Comment:
Just checking, is it intentional to have the top and bottom margins of the nested lists change over breakpoints? Only asking since the bottom margin of the first level is fixed (line 33)

Reply:
just checked with Jason, and we want them to be 16px at all breakpoints. I will update :)